guac-install.sh

```diff
@@ -499,8 +501,15 @@ fi
 systemctl enable ${TOMCAT}
 echo
 
-if [ "$installMySQL" = true ]; then
-    # restart mysql
+if [ "${installMySQL}" = true ]; then
+
+    if [ -e /etc/mysql/mariadb.conf.d/50-server.cnf ]; then
+    elif [ -e /etc/mysql/mariadb.conf.d/50-server.cnf ]; then
+    else
+    fi
+
+
```

Comment:
What happened here?

Reply:
Doh!
This wasn't  meant to be in this commit (miss staged it).

This was meant to be in the next commit to address ~ 
 https://issues.apache.org/jira/browse/GUACAMOLE-760